stable/rabbitmq-ha/templates/secret.yaml

```diff
@@ -19,12 +19,12 @@ data:
   {{ else }}
   rabbitmq-erlang-cookie: {{ randAlphaNum 32 | b64enc | quote }}
   {{ end }}
-{{- if .Values.rabbitmqCert.enabled }}
+{{- if and .Values.rabbitmqCert.enabled (or .Values.rabbitmqCert.cacertfile .Values.rabbitmqCert.certfile .Values.rabbitmqCert.keyfile) }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "rabbitmq-ha.fullname" . }}-cert
+  name: {{ .Values.rabbitmqCert.secretName | default (printf "%s-cert" (include "rabbitmq-ha.fullname" . )) }}
```

Comment:
This doesn't make sense. There's no need to create the secret and it would probably fail anyways, since `rabbitmqCert.secretName` refers to an existing secret. I'd suggest you rename `rabbitmqCert.secretName` to `rabbitmqCert.existingSecret`.